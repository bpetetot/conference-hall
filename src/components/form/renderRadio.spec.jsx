/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import RenderRadio from './renderRadio'

describe('components/renderRadio', () => {
  it('should render', () => {
    const props = { input: { value: 'value', name: 'name' }, label: 'label' }
    const wrapper = shallow(<RenderRadio {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
