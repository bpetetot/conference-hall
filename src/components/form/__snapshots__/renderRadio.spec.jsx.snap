// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/renderRadio should render 1`] = `
<RadioBox
  checked={false}
  name="name"
  value="value"
>
  label
</RadioBox>
`;
